drivers/SmartThings/matter-switch/fingerprints.yml

```diff
@@ -4,8 +4,8 @@ matterManufacturer:
     deviceLabel: THIRDREALITY Night Light
     vendorId: 0x1407
     productId: 0x1088
-    deviceProfileName: light-color-level-illuminance-motion-1000K-15000K 
-    
+    deviceProfileName: light-color-level-illuminance-motion-1000K-15000K
```

Comment:
Just a note, the extraneous changes in this file are just cleaning up extra whitespace and will not affect the fingerprints